src/maintainer/knowledge_base.rst

```diff
@@ -791,6 +793,52 @@ Following implies that ``python`` is a runtime dependency and a Python matrix fo
 
 You need to rerender the feedstock after this change.
 
+Using CMake
+===========
+
+`CMake <https://cmake.org/>`__ can be used to build more complex projects in ``build.sh``
+or ``bld.bat`` scripts.
+
+If you are using cmake, be sure to make it a build requirement in the ``build`` or ``host`` section.
+
+.. code-block:: yaml
+
+    host:
```

Comment:
```suggestion
    build:
```

Reply:
Ok, I can change this -- although as I understand it, `host` by itself should be preferable to a separate `build` and `host` section. Am I mistaken?

Reply:
Yes, `cmake` in `host` is wrong. `host` is usually headers, libraries, host. `build` is build tools like the compiler, make, cmake, pkg-config.